vtex-apps/add-to-cart-button
# Add item property productId to pixel events

#### What does this PR do? \*

Add new property to pixel event `productId`

#### How to test it? \*

1. Open https://breno2--storecomponents.myvtex.com/blouse/p
2. Add the product to cart
3. Check the variable `pixelManagerEvents` in dev tools


Check if `productId` is present in the events `addToCart`, `removeFromCart`, and `cartChanged`:

![image](https://user-images.githubusercontent.com/284515/83452925-f5a6f800-a42f-11ea-9f4b-baf84b51cbdb.png)



#### Describe alternatives you've considered, if any. \*

n/a

#### Related to / Depends on \*
